etc/chromeos/lib/installs.sh

```diff
@@ -322,3 +368,20 @@ install_sqlx_prepare_deps() {
     cargo install sqlx-cli --no-default-features --features postgres,rustls
     sudo apt install --yes netcat # used for `nc`
 }
+
+install_tc_redirect_tap() {
```

Comment:
Since this requires golang, I'm not making this mandatory. 
I'm split on whether it makes sense to auto-install golang or just package this up somewhere for downloading in the future

Reply:
Packaging it and downloading from, say, Cloudsmith probably makes the most sense long-term.

Short-term, you could compile it in a Go container and just spit the compiled binary out.